.gitignore

```diff
@@ -54,6 +54,7 @@ develop-eggs
 lib
 lib64
 pip-wheel-metadata
+*.whl
```

Comment:
Please don't change this file in this PR. If the change is needed, please make a separate PR with it.

Reply:
Sorry, I did it to test locally with the api by manually copying and installing the wheel package locally.
I removed this change from this PR.